src/test/java/org/apache/sling/jcr/contentloader/it/ContentloaderTestSupport.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.sling.jcr.contentloader.it;
 
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static org.apache.felix.hc.api.FormattingResultLog.msHumanReadable;
 import static org.apache.sling.testing.paxexam.SlingOptions.awaitility;
```

Comment:
Remove this if using the custom ContentloaderTestSupport#awaitility method instead.